_posts/2022-07-11-announcing-online-talks.md

```diff
@@ -0,0 +1,34 @@
+---
+author: DjangoCon US Organizers
+category: General
+date: 2022-07-11 06:00:00
+layout: post
+image: /static/img/blog/speaking.jpg
+post_photo_url: /static/img/blog/speaking.jpg
+post_photo_alt: "Speaker addressing a crowd at DjangoCon US 2019 in San Diego"
+title: "Announcing Our DjangoCon US 2022 Online Talks!"
+---
+
+We are delighted to present our online talk lineup!
+
+During the breaks in the in-person sessions (lunch, morning/afternoon breaks, etc.), we will air some online exclusive talks. In-person attendees will get the links to the videos upon the conclusion of talks on Wednesday, October 19.
```

Comment:
> online exclusive talks 

Maybe "talks exclusive to our online-only ticket holders"?

Reply:
works for me